vmicore/src/lib/os/linux/ActiveProcessesSupervisor.cpp

```diff
@@ -69,6 +83,14 @@ namespace VmiCore::Linux
         processInformation->name = *vmiInterface->extractStringAtVA(taskStruct + vmiInterface->getOffset("linux_name"),
                                                                     vmiInterface->convertPidToDtb(SYSTEM_PID));
 
+        // Special case: The process with pid 0 is a kthread and therefore has got no mm_struct. In this case we simply
```

Comment:
The comment is not quite right, see https://elixir.bootlin.com/linux/v5.0.21/source/kernel/sched/core.c#L5372.
However, code is fine I guess.

Reply:
Fixed